spec/05-migration/plugins/opentelemetry/migrations/001_331_to_332_spec.lua

```diff
@@ -0,0 +1,61 @@
+
+local cjson = require "cjson"
+local uh = require "spec.upgrade_helpers"
+
+local v = os.getenv("OLD_KONG_VERSION"):sub(1,5):gsub("%.", "");
+local handler = tonumber(v) >= 330 and describe or pending
```

Comment:
Can you make this into a helper instead?  Otherwise the code will end up be copy-pasted into the next test.